sql/sys_vars.cc

```diff
@@ -6397,7 +6397,7 @@ static Sys_var_charptr Sys_wsrep_data_home_dir(
 
 static Sys_var_charptr Sys_wsrep_cluster_name(
        "wsrep_cluster_name", "Name for the cluster",
-       READ_ONLY GLOBAL_VAR(wsrep_cluster_name), CMD_LINE(REQUIRED_ARG),
+       READ_ONLY PREALLOCATED GLOBAL_VAR(wsrep_cluster_name), CMD_LINE(REQUIRED_ARG),
```

Comment:
Why this change is needed?

Reply:
For preventing memory leak.

Reply:
If `PREALLOCATED` is set, variable will be freed, this is configured in that way for other `wsrep_*` variables.